float64 west_to_east_position
float64 south_to_north_position